.github/workflows/acceptance.yml

```diff
@@ -248,9 +258,16 @@ jobs:
           save-only: true
           snapshot-path: .artifacts/visual-snapshots
 
+      # This job runs when FE or BE changes happen. This is because FE changes could affect the
+      # set up for running the acceptance tests.
+      # We only want to upload code coverage data for BE changes since the FE changes do not
+      # affect the code coverage score
       - name: Handle artifacts
         uses: ./.github/actions/artifacts
         if: needs.files-changed.outputs.backend_all == 'true'
+        with:
+          files: .artifacts/*.coverage.xml
+          type: backend
```

Comment:
@evanpurkhiser Hey Evan, what source code paths are tested when we run visual snaptshots for chartcuterie?